packages/shorebird_cli/test/src/commands/patch/android_patcher_test.dart

```diff
@@ -436,6 +441,60 @@ Looked in:
           );
 
           expect(result, hasLength(Arch.values.length));
+          for (final bundle in result.values) {
+            expect(bundle.hashSignature, isNull);
+          }
+        });
+
+        group('when a private key is provided', () {
+          const testPrivateKeyContent = '''
+-----BEGIN PRIVATE KEY-----
+key
+-----END PRIVATE KEY-----
+''';
```

Comment:
I don't think we care about the content of the private key file, just that it exists, right? I think we could get away with writing an empty file, which would make the tests a bit clearer.